taar/recommenders/local_recommender.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+ADDON_LIST_PER_LOCALE_URL =\
+    "local address"
+    # TODO: this should be a public s3 bucket IFF we are allowed to share this info publically.
+
+def fetch_json(uri):
+    """ Perform an HTTP GET on the given uri, return the results as json.
+    If there is an error fetching the data, raise an exception.
+
+    Args:
+        uri: the string URI to fetch.
+
+    Returns:
+        A JSON object with the response.
+    """
+    data = requests.get(uri)
```

Comment:
This function slightly changed on master. We're not throwing anymore, we're returning `None` if there's any problem with the resource, so that `can_recommend` knows about  it and can properly decide.

You can simply copy that function here (I don't mind too much as this is a prototype for the demo), or you could add that to a shared file and use the latter in both the collaborative filtering model and here. Whatever is quicker ;)